docs/source/cli-reference/help-text-options.rst

```diff
@@ -104,6 +104,8 @@ The Following Help Text is displayed, i.e. This is the help text for Scancode Ve
                                       Files and directories without findings
                                       are omitted (file information is not
                                       treated as findings).
+      --filter-ignoralbe-copyrights   Filter out copyrights that are for the 
```

Comment:
I think we have a tiny typo:
`--filter-ignoralbe-copyrights` should be `--filter-ignorable-copyrights`
```suggestion
      --filter-ignorable-copyrights   Filter out copyrights that are for the 
```